app/components/avatar_component.rb

```diff
@@ -2,13 +2,17 @@
 
 # Renders a User's avatar as image or user's initials
 class AvatarComponent < ApplicationComponent
```

Comment:
It feels weird to have User as a param and Pet as an option. Can they both be options instead for consistency?